### PR TITLE
Style cleanup for `models/dalec/R/*.R` scripts

### PR DESCRIPTION
**Pull request by bpbond**
Ran through formatR::tidy_dir with arrow=TRUE and indent=2; made sure all if blocks have braces; took advantage of paste0 and file.path; changed require calls to library.

Note - got rid of extra params (…) in `met2model.DALEC`. They weren’t last, which was odd, and weren’t used.

### CODE REVIEW

File: models/dalec/R/model2netcdf.DALEC.R [hunk 1]
```diff
@@ -21,116 +22,105 @@
...
+    output[[7]] <- (sub.DALEC.output[, 11] * 0.001) / timestep.s  # Woody Litter, kgC/m2/s
+    output[[8]] <- (sub.DALEC.output[, 13] * 0.001) / timestep.s  # Root Litter, kgC/m2/s
     
     ## non-standard variables: Pools
-    output[[9]]  <- (sub.DALEC.output[,15]* 0.001)    # Leaf Biomass, kgC/m2
-    output[[10]] <- (sub.DALEC.output[,17]* 0.001)    # Wood Biomass, kgC/m2
-    output[[11]] <- (sub.DALEC.output[,19]* 0.001)    # Root Biomass, kgC/m2
-    output[[12]] <- (sub.DALEC.output[,27]* 0.001)    # Litter Biomass, kgC/m2
-    output[[13]] <- (sub.DALEC.output[,29]* 0.001)    # Soil C, kgC/m2
+    output[[9]] <- (sub.DALEC.output[, 15] * 0.001)  # Leaf Biomass, kgC/m2
```
Comment (mdietze) [inline models/dalec/R/model2netcdf.DALEC.R]: jagged

File: models/dalec/R/met2model.DALEC.R [hunk 2]
```diff
@@ -75,103 +80,99 @@ met2model.DALEC <- function(in.path, in.prefix, outfolder, start_date, end_date,
...
-    lat  <- ncvar_get(nc,"latitude")
-    lon  <- ncvar_get(nc,"longitude")
-    Tair <- ncvar_get(nc,"air_temperature")  ## in Kelvin
-    SW   <- ncvar_get(nc,"surface_downwelling_shortwave_flux_in_air") ## in W/m2
-    CO2  <- try(ncvar_get(nc,"mole_fraction_of_carbon_dioxide_in_air"))
+    lat <- ncvar_get(nc, "latitude")
+    lon <- ncvar_get(nc, "longitude")
+    Tair <- ncvar_get(nc, "air_temperature")  ## in Kelvin
+    SW <- ncvar_get(nc, "surface_downwelling_shortwave_flux_in_air")  ## in W/m2
+    CO2 <- try(ncvar_get(nc, "mole_fraction_of_carbon_dioxide_in_air"))
```
Comment (bpbond) [inline models/dalec/R/met2model.DALEC.R]: Agreed. Happy to un-fix?

File: models/dalec/R/met2model.DALEC.R [hunk 3]
```diff
@@ -24,32 +24,37 @@
 ##' @param end_date the end date of the data to be downloaded (will only use the year part of the date)
 ##' @param overwrite should existing files be overwritten
 ##' @param verbose should the function be very verbose
-met2model.DALEC <- function(in.path, in.prefix, outfolder, start_date, end_date, ..., overwrite=FALSE,verbose=FALSE){
+met2model.DALEC <- function(in.path, in.prefix, outfolder, start_date, end_date,
+                            overwrite = FALSE, verbose = FALSE) {
```
Comment (bpbond) [inline models/dalec/R/met2model.DALEC.R]: Thanks. I'll fix.

**Comment by mdietze** [comment]
Just needs an "Update branch"

**Pull request merged**